src/features/auth/services/use-register.ts

```
@@ -0,0 +1,24 @@
+import { api } from '@/features/core';
+
+interface RegisterResponse {
+    accessToken: string;
+}
+
+export const register = async ({
+    email,
```

ESLint

[error] Delete `··`

(prettier/prettier)